softlayer/vm/interface.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 type VMCloudProperties struct {
+	VmNamePrefix             string                               `json:"vmNamePrefix,omitempty"`
```

Comment:
Ok, in that case we need to switch back everything that was changed. Currently it is not passing integration tests. I'll see if I can switch it back everywhere and submit a pull request.